app/src/main/res/values-pl/strings.xml

```diff
@@ -1,47 +1,113 @@
 <resources>
     <string name="channel_name">Pokaż sterowanie multimediami</string>
     <string name="channel_description">Kanał powiadomień do wyświetlania elementów sterujących mediami.</string>
+    <string name="button_pause">Pauza</string>
+    <string name="button_play">Odtwórz</string>
+    <string name="button_backward">Do tyłu</string>
+    <string name="button_forward">Do przodu</string>
+    <string name="button_bookmark">Zakładka</string>
 
     <!-- Toast messages -->
     <string name="no_albums">Nie znaleziono żadnych albumów w podanym katalogu.</string>
+    <string name="no_directories">Nie dodano jeszcze żadnych folderów.</string>
     <string name="no_audio_files">Nie znaleziono żadnych plików audio w tym albumie.</string>
-    <string name="play_error">Wybrany plik audio nie nie może być odtwarzany. Możliwe, że ten plik już nie istnieje.</string>
+    <string name="play_error">Wybrany plik audio nie może być odtwarzany. Możliwe, że ten plik już nie istnieje.</string>
     <string name="permission_denied">Nie udzielono niezbędnej zgody. Zamknięto AudioAnchor.</string>
-    <string name="write_permission_denied">Nie udzielono niezbędnej zgody. Eksport nie może zostać wykonany.</string>
+    <string name="write_permission_denied">Nie udzielono niezbędnej zgody. Nie można zapisywać w pamięci.</string>
     <string name="time_format_error">Proszę wpisać prawidłowy czas w formacie hh:mm:ss.</string>
     <string name="import_success">Importowanie udane :)</string>
     <string name="import_fail">Coś poszło nie tak. Import nie powiódł się.</string>
-    <string name="export_success">Wyeksportowano bazę danych do %s</string>
+    <string name="export_success">Wyeksportowano bazę danych do %s.</string>
     <string name="export_fail">Coś poszło nie tak. Eksport nie powiódł się.</string>
-    <string name="synchronize_success">Zsynchronizowano katalog.</string>
-    <string name="audio_file_error">Wystąpił problem z plikami audio \"%s\".</string>
+    <string name="synchronize_success">Zsynchronizowano bibliotekę.</string>
+    <string name="audio_file_error">Wystąpił problem z plikiem audio \"%s\".</string>
     <string name="empty_title_error">Proszę wpisać tytuł swojej zakładki.</string>
-    <string name="jumped_to_bookmark">Skoczono do zakładki \"%s\" w %s</string>
+    <string name="jumped_to_bookmark">Skoczono do zakładki \"%s\" w %s.</string>
     <string name="bookmark_deleted">Usunięto zakładkę.</string>
-    <string name="tracks_deleted_from_db">Usunięto %d ścieżkę(-ki) dźwiękową(-wych) z bazy danych.</string>
+    <string name="bookmark_added_toast">Dodano zakładkę \"%s\" w %s.</string>
+    <plurals name="tracks_deleted">
+        <item quantity="one">Usunięto %d ścieżkę dźwiękową.</item>
+        <item quantity="few">Usunięto %d ścieżki dźwiękowe.</item>
+        <item quantity="other">Usunięto %d ścieżek dźwiękowych.</item>
```

Comment:
I get the error message for this and all other plural strings that for Polish the quantity *many* should also be defined. I don't know any Polish, so I don't know if it makes any difference in this scenario. Was there a reason you didn't add it?

Reply:
`other` replaces `zero` and `many` here, so `other` is enough here.